src/components/UiCardItem/SmallCarditem.tsx

```diff
@@ -1,33 +1,27 @@
 import { Stack } from '@mui/material';
-import Image from 'next/image';
 import React from 'react';
 import { Trans } from 'react-i18next';
 
 import { HoverCard } from '../../features/landing/components/Possibilities/HoverCard';
-import { UiTypography } from '../ui';
-import { UiTooltip } from '../ui/UiTooltip';
+import UiImage from '../UiImage';
+import UiTooltip from '../UiTooltip';
+import UiTypography from '../UiTypography';
 
-import { cardItemPossibilitiesStyles } from './styles';
-import styles from './styles.module.scss';
+import { smallCarditemStyles } from './styles';
 
 function SmallCarditem({ item, imageList }: any) {
   return (
-    <Stack sx={cardItemPossibilitiesStyles.wrapper}>
-      <Image
+    <Stack sx={smallCarditemStyles.wrapper}>
+      <UiImage
         src={item.imageSrc}
         alt="Card Image"
```

Comment:
i see a lot of places where you use alt like this, change it to localization in all application